### ISSUE TITLE
Make it work with Node.js

### ISSUE THREAD

**Issue by ricordisamoa** [wikimedia/jquery.i18n#83]
Especially some components like the parser would be amazingly useful server-side.

**Comment by santhoshtr** [comment]
I agree on this. I was thinking about this for a while. There is some DOM manipulation related stuff in this library -especially the one that change the elements text and using data-i18n, but rest of the core code does not need any jquery.

But I am not committing to work on this immediately. So expect delay unless some volunteer help :)

**Comment by ricordisamoa** [comment]
Yay! Let me know how you can be helped :)

**Comment by amire80** [comment]
I'm thinking of working on it as a pet project.

It would be useful for my Telegram bot project: https://github.com/amire80/mediawiki-telegram-bot . That project is also a pet project, so I am not committing to anything, and other volunteers are still very welcome to help :)

**Comment by santhoshtr** [comment]
+cc @Mooeypoo

**Comment by santhoshtr** [comment]
I am working on this again - to build a generic, modern js library https://github.com/santhoshtr/banana-i18n

Please take a look.
